src/types/Fradrag.ts

```diff
@@ -1,7 +1,7 @@
 import { Nullable } from '~lib/types';
 
 export interface Fradrag {
-    type: Fradragstype;
+    type: Fradragstype | 'Forventet inntekt';
```

Comment:
Syns det er litt tricky å bruke en string literal-type her (og rundt omkring ellers i applikasjonen).

Kanskje vi kunne prøvd noe ala å definere disse:
```ts
export const ForventetInntektfradrag = 'Forventet inntekt' as const;
export type ForventetInntektfradrag = typeof ForventetInntektfradrag;
```

og så bruke dem slik:

```suggestion
    type: Fradragstype | ForventetInntektfradrag;
```